addons/core/stringtable.xml

```diff
@@ -1171,139 +1193,164 @@
             </Key>
             <Key ID="STR_TFAR_SETTINGS_volumeIndicatorTransparency">
                 <English>Volume Indicator Transparency</English>
                 <Chinese>音量指示器透明度</Chinese>
                 <Chinesesimp>音量指示器透明度</Chinesesimp>
+                <Japanese>音量インジケータの透明度</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_volumeIndicatorTransparency_desc">
                 <English>Volume Indicator Transparency (0 fully visible - 1 invisible)</English>
                 <Chinese>音量指示器透明度 (0為完全透明，1為不透明)</Chinese>
                 <Chinesesimp>音量指示器透明度 (0为完全透明，1为不透明)</Chinesesimp>
+                <Japanese>音量インジケータの透明度を決定します (0 で表示 - 1 で透明)</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_useOldVolumeHint">
                 <English>Use old Voice Volume Hint</English>
                 <Chinese>使用舊的語音音量提示</Chinese>
                 <Chinesesimp>使用旧的语音音量提示</Chinesesimp>
+                <Japanese>旧音量ヒントを使う</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_useOldVolumeHint_desc">
                 <English>Use old Voice Volume Hint</English>
                 <Chinese>使用舊的語音音量提示</Chinese>
                 <Chinesesimp>使用旧的语音音量提示</Chinesesimp>
+                <Japanese>古い音量ヒントを使用します</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_showTransmittingHint">
                 <English>Show Transmitting Hint</English>
                 <Chinese>顯示無線電發送訊息提示</Chinese>
                 <Chinesesimp>显示无线电发送讯息提示</Chinesesimp>
+                <Japanese>送信ヒントを表示</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_showTransmittingHint_desc">
                 <English>Show the Hint (Bottom right) when you are Transmitting</English>
                 <Chinese>當無線電傳送訊息時在右下角顯示提示訊息</Chinese>
                 <Chinesesimp>当无线电传送讯息时在右下角显示提示讯息</Chinesesimp>
+                <Japanese>送信中はヒント (右下) を表示します</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_showChannelChangedHint">
                 <English>Show Channel Changed Hint</English>
                 <Chinese>顯示無線電頻道更改提示</Chinese>
                 <Chinesesimp>显示无线电频道更改提示</Chinesesimp>
+                <Japanese>チャンネル変更ヒントを表示</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_showChannelChangedHint_desc">
                 <English>Show the Hint (Bottom right) when you switch Channels via UI or Hotkey</English>
                 <Chinese>當無線電頻道更改時在右下角顯示提示訊息</Chinese>
                 <Chinesesimp>当无线电频道更改时在右下角显示提示讯息</Chinesesimp>
+                <Japanese>UI か割り当てキーでチャンネルを変更するとヒント (右下) を表示します</Japanese>
             </Key>
             <Key ID="STR_TFAR_SETTINGS_FullDuplex">
                 <English>Full-Duplex</English>
                 <French>Duplex complet</French>
                 <German>Voll-Duplex</German>
                 <Japanese>全二重通信</Japanese>
                 <Korean>양방향 통신</Korean>
-                <English>Full-Duplex</English>
```

Comment:
Shouldn't stringtables have a <original> entry?

Reply:
this removal is fine.
If comes from converting original to english

Reply:
I know it's fine. But what does Arma do if there is no <original> entry? Does it automatically use english or the first entry or at random?

Reply:
the first entry